rcl/src/rcl/subscription.c

```diff
@@ -224,6 +229,8 @@ rcl_take(
     RCL_SET_ERROR_MSG(rmw_get_error_string_safe(), options->allocator);
     return RCL_RET_ERROR;
   }
+  RCUTILS_LOG_DEBUG_NAMED(
+    ROS_PACKAGE_NAME, "Subscription take request succeeded: %s", taken ? "true" : "false")
```

Comment:
copy-n-paste ?

Reply:
fixed in 88b8bee2b4665cc5ed9b9846f6b17dc69aaf4ebd thanks